README.txt

```diff
@@ -7,17 +7,19 @@ Target: Run smoothly as possible even on laggy server
 
  Features
 ----------
-- A fast cart for your railway or roller coaster (up to 10 m/s!)
+- A fast cart for your railway or roller coaster!
 - Boost and brake rails
 - By mesecons controlled Start-Stop rails
 - Detector rails that send a mesecons signal when the cart drives over them
 - Rail junction switching with the 'right-left' walking keys
 - Handbrake with the 'back' key
 
 
- License for everything
+ Copying / Copyright
 ------------------------
```

Comment:
Could you please also adjust the amount of dashes here? (length of title + 2 in total)